app/javascript/lib/validation/validate-file-content.js

```diff
@@ -348,15 +403,25 @@ async function parseFile(file, fileType) {
       if (lines.length < 2) {
         return { table, delimiter, issues: [['error', 'cap:format:no-newlines', 'File does not contain newlines to separate rows']] }
       }
+    }
+    if (['Cluster', 'Metadata', 'Expression Matrix'].includes(fileType)) {
       // if there are no encoding issues, and this isn't a gzipped file, validate content
       delimiter = sniffDelimiter(headerLines, mimeType)
       table = []
       for (let i = 0; i < lines.length; i++) {
         table.push(lines[i].split(delimiter))
       }
-      const headerTable = table.slice(0, 2)
-      issues = await validateCapFormat(headerTable, fileType)
-      issues = issues.concat(validateUniqueCellNamesWithinFile(table))
+
+      if (fileType === 'Expression Matrix') {
+        const firstRowTable = table.slice(0, 1)
+        issues = issues.concat(validateUniqueCellNamesWithinFile(firstRowTable[0], fileType))
+        issues = issues.concat(validateGeneInHeader(firstRowTable))
+        issues = issues.concat(validateColumnNumberAndValues(table))
```

Comment:
Blocker: this validation seems relevant for cluster and metadata files, too.  See my comments in that function for detail.

```suggestion
        issues = issues.concat(validateColumnNumberAndValues(table))
```